assets/js/googlesitekit/widgets/components/URLSearchWidget.js

```diff
@@ -45,9 +48,10 @@ function URLSearchWidget( { Widget } ) {
 	const { navigateTo } = useDispatch( CORE_LOCATION );
 	const onClick = useCallback( () => {
 		if ( match?.permalink ) {
+			trackEvent( viewContext, 'open_urldetails' );
```

Comment:
@johnPhillips let's wait till the trackEvent function is executed and only then navigate to the details URL.

Reply:
Ah yes, thanks. I only just realised `trackEvent` can be awaited. Good call.